### ISSUE TITLE
C* preferred heap size settings

### ISSUE THREAD

**Issue by benbromhead** [instaclustr/cassandra-operator#99]
On an 8Gi pod claim, we are setting heap size to 2GB. Is this optimal? I know we need to leave space for offheap but this is probably a bit too conservative. Maybe minimum recommended pod size should be 16GB (8 miniumum for heap, 8 for offheap)

INFO  [main] 2018-10-08 17:52:27,082 CassandraDaemon.java:481 - Heap size: 2.000GiB/2.000GiB

**Comment by zegelin** [comment]
Yes, these could be tuned better. See https://github.com/instaclustr/cassandra-operator/blob/master/java/operator/src/main/java/com/instaclustr/cassandra/operator/controller/DataCenterReconciliationController.java#L324

That logic should be similar to the default `cassandra-env.sh`, but this should be confirmed.
